pages/docs/concepts/asyncapi-document/add-server.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Add servers
+weight: 295
+---
+
+A server represents a message broker handling communication between producers and consumers. Adding and defining servers is useful because it specifies where and how to connect. The connection facilitates where to send and receive messages.
+
+A unique key identifies the server and contains information about the server's connection details, such as the URL, protocol, and authentication. The diagram below describes how to define and add servers in AsyncAPI.
+
+```mermaid
+graph TD
+  subgraph AsyncAPI
+    A[Channel: userActions]
+    B[Operation: subscribe]
+    C[Message]
+    D[Payload]
+  end
+
+  subgraph Servers
+    SA[Servers: development, production]
+  end
+
+style A fill:#47BCEE,stroke:#47BCEE;
+style B fill:#47BCEE,stroke:#47BCEE;
+style C fill:#47BCEE,stroke:#47BCEE;
+style SA fill:#47BCEE,stroke:#47BCEE;
+style D fill:#47BCEE,stroke:#47BCEE;
+
+  SA --> A
+
+  A -->|inherits servers| B
+  B -->|references| C
+  C -->|references| D
```

Comment:
```suggestion
graph TD
    A[channel: userActions]
    B[operation: subscribe]
    C[message]
    D[payload]
    SA[servers: development, production]

style SA fill:#47BCEE,stroke:#47BCEE;

  SA -->|AsyncAPI| A
  A -->|inherits servers| B
  B -->|references| C
  C -->|references| D
```

<img width="900" src="https://github.com/asyncapi/website/assets/19964402/34523c2a-24a5-4893-9719-b5a56250307c">

@kakabisht, a few feedback points on your diagrams to ensure consistency across all project diagrams:
-  **Use only nodes and remove subgraphs** since we're not really doing that across our docs. Additionally, I want you to notice that removing the subgraphs fixes the bug I noticed in the comment https://github.com/asyncapi/website/pull/1927/files#r1396555536. 
- don't capitalize the components because we don't do that in our spec
- We're only highlighting the featured component in our diagrams with the blue hex code. I believe this means your diagram should only highlight the server node. 

🔗 [LIVE mermaid link to my proposed diagram changes](https://mermaid-js.github.io/mermaid-live-editor/edit#pako:eNplkD9rwzAQxb-KUFdnKxRUKMh2hm6FdLM6KNI5FtE_TlLAxPnulROnSzXd-72nu-OuVAUNlNETyjiR7154Uh8f1CS9B8tISYBcZRN8-nmY7RAioFwRI6kck0JzhM3sBgcpydNT90OUsw1Sb_rAh9rwApgY0XABG6IDnxsSMehyn7MlhU95tlB_kNFYy15e39puv29SxnCGp3wXfo3X0G73sfA0e8W_PhfCV_qAxk-AJieyDV5Iu5rt3UQYAcErqLhbcfcP97ShDtBJo-uhrmtI0DyBA0FZLTWMstgsqPC3GpUlh0Ndg7KMBRpaopYZeiPriR1lo7Tpj-61yQE3ePsFcWuEvw )
_cc @derberg in case he disagrees with my proposed diagram_